### PR TITLE
PHP CodeSniffer

### PR DESCRIPTION
**Pull request by angelxmoreno**
- added PHPCS to composer
- added helper bash script to report styles
- applied styles fixes

### CODE REVIEW

File: bin/checkstyle.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -x
```
Comment (Jwisnewski) [inline bin/checkstyle.sh:2]: wtf is this?

**Review by Jwisnewski — COMMENTED** [review]
please tell me what x does

File: bin/checkstyle.sh (hunk 1 above)
Comment (angelxmoreno) [reply bin/checkstyle.sh:2]: not sure. i always copy and pasta

Comment (angelxmoreno) [reply bin/checkstyle.sh:2]: ## 2.3.2. Debugging on part(s) of the script
```
--
set -x			# activate debugging from here
w
set +x			# stop debugging from here
--
```

Found some good stuff here : http://tldp.org/LDP/Bash-Beginners-Guide/html/sect_02_03.html

Comment (Jwisnewski) [reply bin/checkstyle.sh:2]: cool thanks I get it now

**Review by Jwisnewski — APPROVED** [review]

**Pull request merged**